docs/rules/control-character-escape.md

```diff
@@ -0,0 +1,44 @@
+---
+pageClass: "rule-details"
+sidebarDepth: 0
+title: "regexp/control-character-escape"
+description: "enforce consistent escaping of control characters"
+---
+# regexp/control-character-escape
+
+> enforce consistent escaping of control characters
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+- :wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.
+
+## :book: Rule Details
+
+This rule reports ???.
```

Comment:
Could you add rule description? Other than that, it looks good!

Reply:
Thank you for pointing that out!